backends/platform/psp/psp.spec

```diff
@@ -1,3 +1,3 @@
 %rename lib	old_lib
 *lib:
--lz -lstdc++ --start-group -lc %:if-exists(libpthreadglue.a%s) %:if-exists(libpthread.a%s) %:if-exists(libcglue.a%s) --end-group -lm -lpspprof -lpspvfpu -lpspdebug -lpspgu -lpspge -lpspdisplay -lpspctrl -lpspsdk -lpsputility -lpspuser -lpsppower -lpsphprm  -lpsprtc -lpspaudio -lpspaudiocodec -lpspkernel -lpspnet_inet %(old_lib)
+-lstdc++ -lpspprof -lpspvfpu -lpspdebug -lpspgu -lpspge -lpspdisplay -lpspctrl -lpsppower -lpsphprm -lpspaudio -lpspaudiocodec %(old_lib)
```

Comment:
I added all these libraries because I tried to keep compatibility with old toolchains not knowing what @rsn8887 would use.
Not sure we want to break this.

Reply:
I have been using the latest Docker toolchain always for a long time. I think these changes are great. I'd rather fully commit to the new toolchain now.